todo/android/todo/src/main/java/com/mongodb/stitch/sdk/examples/todo/MainActivity.java

```diff
@@ -327,7 +324,7 @@ public void onComplete(@NonNull final Task<Void> task) {
     }
 
     private Task<Void> refreshList() {
-        return getItemsCollection().find(new Document("owner_id", _client.getAuth().getUserId())).continueWithTask(new Continuation<List<Document>, Task<Void>>() {
+        return getItemsCollection().find(new Document("owner_id", _client.getUserId()), 100).continueWithTask(new Continuation<List<Document>, Task<Void>>() {
```

Comment:
[style] This likely goes over the column limit for characters

Reply:
👍